model_demos/cv_demos/dla/utils/model.py

```diff
@@ -0,0 +1,553 @@
+from os.path import join
```

Comment:
We can't keep full model implementation here as is. We need to reference the source of, or find another source (e.g. transformers, torchhub, etc.)